nextflow.config

```diff
@@ -91,14 +91,18 @@ profiles {
     includeConfig 'conf/binac.config'
   }
   cfc {
-    includeConfig 'conf/base.config'
```

Comment:
Why did you remove this config file from this profile?

Reply:
Yes, this looks like it's a mistake..?

Reply:
Ooops. Sorry. It's base.config

Reply:
Just checked on this. My forked version has the following snippet
`binac {
    includeConfig 'conf/base.config'
    includeConfig 'conf/binac.config'
  }`